scripts/update

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e
+
+# Start Docker container
+echo "==> Starting Docker container..."
+docker-compose up -d
+
+# Install dependencies in Docker container
+echo "==> Installing dependencies..."
+docker-compose exec web npm install
```

Comment:
i think this should be
```
docker-compose exec app npm install
```
for our purpose, based on the service names in [docker-compose.yml](https://github.com/CodeForPhilly/third-places/blob/develop/docker-compose.yml#L3). likewise for the other `exec` commands below